index.html

```diff
@@ -291,6 +291,31 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer>
+      <div class="form-container">
+        <form action="https://formspree.io/f/mwkaywog" method="post">
+          <h1 class="form-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+          <div class="form-group">
+            <div class="form-control">
+              <label>
+                  <input type="text" name="full_name" placeholder="Full name" required>
```

Comment:
[OPTIONAL] Please, kindly consider limiting the length of the input up to 30 via adding the input attribute `max length` since it was required by the project.